caesars-palace/blocks/form/form.css

```diff
@@ -81,42 +81,46 @@ main .form.sign-in label {
   font-weight: 400;
 }
 
-main .form span.input-error {
+.form span.input-error {
   display: none;
   font-size: 11px;
   color: #d60000;
   line-height: 1.5;
 }
 
-main .form span.input-error.active {
+.form span.input-error.active {
   display: flex;
 }
 
-main .form.sign-in label.active {
+.form.sign-in label.active {
   color: #000;
   transform: scale(0.71) translateY(-50%) translateY(-11px);
 }
 
-main .form label.required::after {
+.form label.required::after {
   content: "*";
   color: red;
   padding-left: 5px;
 }
 
-main .form.form.sign-in label.required::after {
+.form.form.sign-in label.required::after {
   display: none;
 }
 
-main .form .field-wrapper {
+.form .field-wrapper {
   margin-bottom: 20px;
 }
 
-main .form .form-checkbox-wrapper {
+.form.sign-in .field-wrapper {
+  margin-bottom: 16px;
```

Comment:
Use the CSS variable here if you can

Reply:
I don't see an applicable variable for this.